### PR TITLE
*: remove the t.Run

### PR DESCRIPTION
**Pull request by hawkingrei**
Signed-off-by: Weizhen Wang <wangweizhen@pingcap.com>

### What problem does this PR solve?

Issue Number: ref #31716

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: domain/db_test.go [hunk 1]
```diff
@@ -25,8 +25,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// SubTestDomainSession is batched in TestDomainSerial
-func SubTestDomainSession(t *testing.T) {
+// TestDomainSession is batched in TestDomainSerial
```
Comment (LittleFall) [inline domain/db_test.go:28]: this comment should be removed

File: domain/domain_test.go [hunk 2]
```diff
@@ -45,8 +45,8 @@ import (
 	"go.etcd.io/etcd/integration"
 )
 
-// SubTestInfo is batched in TestDomainSerial
-func SubTestInfo(t *testing.T) {
+// TestInfo is batched in TestDomainSerial
```
Comment (LittleFall) [inline domain/domain_test.go:48]: ditto

File: domain/domain_test.go [hunk 3]
```diff
@@ -154,8 +154,8 @@ func SubTestInfo(t *testing.T) {
 	require.NoError(t, failpoint.Disable("github.com/pingcap/tidb/domain/infosync/FailPlacement"))
 }
 
-// SubTestDomain is batched in TestDomainSerial
-func SubTestDomain(t *testing.T) {
+// TestDomain is batched in TestDomainSerial
```
Comment (LittleFall) [inline domain/domain_test.go:157]: ditto

**Review by LittleFall — APPROVED** [review]
others LGTM

File: domain/db_test.go (hunk 1 above)
Comment (hawkingrei) [reply domain/db_test.go]: removed

File: domain/domain_test.go (hunk 2 above)
Comment (hawkingrei) [reply domain/domain_test.go]: removed

File: domain/domain_test.go (hunk 3 above)
Comment (hawkingrei) [reply domain/domain_test.go]: removed

**Review by ngaut — APPROVED** [review]

**Pull request merged**